### ISSUE TITLE
Info: check technical

### ISSUE THREAD

**Issue by ErwinKomen** [ErwinKomen/RU-passim#762]
> ... reviewen of de technische informatie op de Home page correct is / aangevuld moet worden

**Comment by ErwinKomen** [comment]
### Actions
1. Home page 
    1. This is `index.html`
    2. Double checked and removed superfluous news items
    3. Added item about the upcoming launch
1. The **About** page
    1. This is `about.html`
    2. The statistics, where do they come from (user-input of manuscripts, for instance)?

**Comment by ErwinKomen** [comment]
### Technical information
The 'technical information' on the home page is very concise. I only tweaked the “news items” there.

### About page
The about page still deserves some attention, especially the following items:
1.  Quick reference: should this stay here?
    1. If so: then at least the part about the “main purpose” must be updated
    1. The info about the source code, the link to github, the link to the short guide and manual (pdf) – that's all ok
1. Bibliography:
    1. maybe that section should be removed completely?
    1. Perhaps it should be replaced by “How to quote”
    (Currently, via Help > How to quote you will end up on the About page, where you will not see any information about “quoting”)
1. I have updated the technical page here and there (especially the part about Data access, which talks about visualizations).

### Menu items
Another small point: currently selecting Tools > Visualizations leads to ... the list of used literature.
Suggestions: 
1. either remove Tools > Visualizations
1. or else: let it lead to MyPassim.

**Comment by shariboodts** [comment]
Thanks for this, response to your comments:

Technical information
The 'technical information' on the home page is very concise. I only tweaked the “news items” there.
**> See powerpoint about Homepage, which provides alternative content**

About page
The about page still deserves some attention, especially the following items:
**>Yes, I will rewrite and provide new content**

Quick reference: should this stay here?
If so: then at least the part about the “main purpose” must be updated
The info about the source code, the link to github, the link to the short guide and manual (pdf) – that's all ok
Bibliography:
maybe that section should be removed completely?
Perhaps it should be replaced by “How to quote”
(Currently, via Help > How to quote you will end up on the About page, where you will not see any information about “quoting”)

Technical page
I have updated the technical page here and there (especially the part about Data access, which talks about visualizations).
One dangling issue may be the API access (bottom right of the technical page). Right now this contains the promise: "Accessing data by a different program will be made possible in the near future". Should that stay?
**>Yes**

Menu items
Another small point: currently selecting Tools > Visualizations leads to ... the list of used literature.
**> See powerpoint about Homepage, which provides alternative content**
Suggestions:

either remove Tools > Visualizations
or else: let it lead to MyPassim.
